diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter1/IdentifierNaming.kt

```diff
@@ -216,7 +194,7 @@ class IdentifierNaming(configRules: List<RulesConfig>) : DiktatRule(
                         }) {
                             // For class members also check `@property` KDoc tag.
                             // If we are here, then `variableName` is definitely a node from a class.
-                            variableName.parent(CLASS)!!.findChildByType(KDOC)?.kDocTags()
+                            variableName.parent(CLASS)?:variableName.parent(OBJECT_DECLARATION)?.findChildByType(KDOC)?.kDocTags()
```

Comment:
```suggestion
                            (variableName.parent(CLASS)?:variableName.parent(OBJECT_DECLARATION))?.findChildByType(KDOC)?.kDocTags()
```
?

Reply:
elvis operator has the biggest priority, I guess...

Reply:
oh shit